packaging/opensuse-42.2/snapd.spec

```diff
@@ -122,7 +121,7 @@ export CXXFLAGS
 # apparmor kernel available in SUSE and Debian. The generated apparmor profiles
 # cannot be loaded into a vanilla kernel. As a temporary measure we just switch
 # it all off.
-%configure --disable-apparmor --disable-seccomp --libexecdir=/usr/lib/snapd
```

Comment:
I'm somewhat unhappy about disabling seccomp on openSUSE since it worked OK with it and now it is a step in the wrong direction. I'll tweak this to see what is missing to make it work.